### PR TITLE
Add function to parse table to list

### PR DESCRIPTION
**Pull request by samuelloza**

### CODE REVIEW

File: src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java [hunk 1]
```diff
@@ -218,4 +222,65 @@ public static String getValueOfWebElement(final By by) {
     public static void waitLoadPage(final WebElement webElement) {
```
Comment (MirkoFer-622) [inline src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java:222]: Change the name of this method for other more descriptive, please

File: src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java [hunk 2]
```diff
@@ -218,4 +222,65 @@ public static String getValueOfWebElement(final By by) {
...
         WebDriverManager.getInstance().getWebDriverWait().until(ExpectedConditions.visibilityOf(webElement));
     }
+
+    /**
+     * [SL] Sends the key down simulating a scroll.
+     *
+     * @param scroll
+     */
+    public static void loadDynamicTable(final By scroll) {
```
Comment (MirkoFer-622) [inline src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java:231]: Change the name of this method for another more descriptive, please (it should not be useful only for DynamicTables, true?)

File: src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java [hunk 3]
```diff
@@ -218,4 +222,65 @@ public static String getValueOfWebElement(final By by) {
...
+    /**
+     * [SL] Converts the content of a table to list and every element contain a map key=header, value=text.
+     *
+     * @param headerTable  a By of the header of table
+     * @param contentTable a By of the content of table
+     * @param idRow        a identification unique of every row for example the id
+     * @param attribute    an attribute for get the id
+     * @return a List of all elements of table
+     */
+    public static List<Map<String, String>> parseDynamicTableAsList(final By headerTable, final By contentTable,
```
Comment (MirkoFer-622) [inline src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java:244]: This method is not responsability of this class, please remove it and put it in another one.

**Comment by samuelloza** [comment]
I close this PR because the feature parseTableToList was a discarded

**Pull request closed without merging**